src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```

Comment:
###AWAS4yv3CSWYuT4mbB7d###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_21@6
Complete the task associated to this TODO comment.

Reply:
simulating user comment on about to be deleted comment

Reply:
simulating bot comment not autogenerated

Reply:
let's do another user comment, just because